docs/jwst/master_background/description.rst

```diff
@@ -11,38 +11,62 @@ versus wavelength - is projected into the
 exposure types. The master background subtraction step uses the input members designated
-with "exptype: background" as the data from which to create the master background spectrum.
-These should be :ref:`x1d <x1d>` products created for individual exposures at the end of
-the :ref:`calwebb_spec2 <calwebb_spec2>` pipeline.
-The master background will be subtracted from all input members designated as
-"exptype: science" in the ASN, resulting in a new version of each such input. These inputs
-should be :ref:`cal <cal>` products created for individual exposures by the
+with "exptype: background" to create the master background spectrum.
+These need to be :ref:`x1d <x1d>` products created from individual exposures at the end of
+the :ref:`calwebb_spec2 <calwebb_spec2>` pipeline, containing spectra of background regions.
+The master background signal will be subtracted from all input members designated as
+"exptype: science" in the ASN, resulting in a new version of each science input. These inputs
+need to be :ref:`cal <cal>` products created from individual exposures by the
 :ref:`calwebb_spec2 <calwebb_spec2>` pipeline.
 
-For observing scenarios that use exposures of dedicated background targets, the "background"
-members in the ASN will be the exposures from those background targets. For scenarios that
-use source nodding within the field to sample the background, the lists of "science" and
-"background" members in the ASN will use the same exposures. The only difference is that
-the "science" members will refer to :ref:`cal <cal>` products, while the "background"
-members will refer to :ref:`x1d <x1d>` products.
+There are two main observing scenarios that are supported by this step: nodded point sources
+and off-source background exposures of extended targets. A third type of operation is performed
+for NIRSpec MOS observations that include background slits. The details for each mode are explained
+below.
 
-For example, the ASN file for a simple 2-point nodded observation consisting of two
+Nodded Point Sources
+^^^^^^^^^^^^^^^^^^^^
+If an observation uses a nodding type dither pattern to move a small or point-like source within
+the field-of-view, it is assumed that part of the field-of-view in each exposure is also suitable
+for measuring background. Exposures of this type are identified by the pipeline based on their
+"PATTTYPE" (primary dither pattern type) keyword value. The value will either contain the
+substring "NOD" somewhere within the name (e.g. "2-POINT-NOD" or "ALONG-SLIT-NOD"), or will
+be set to "POINT-SOURCE" (for MIRI MRS).  The :ref:`calwebb_spec2 <calwebb_spec2>`
+:ref:`srctype <srctype_step>` step recognizes these PATTTYPE values and sets the
+source type to "POINT".
+
+This in turn causes the :ref:`extract_1d <extract_1d_step>` step at
+the end of :ref:`calwebb_spec2 <calwebb_spec2>` to extract spectra for both source and
+background regions. For IFU exposures the background region is typically an annulus that is
```

Comment:
For IFU data, it is true that a background annulus will be subtracted by default.  For non-IFU data, however, a background will only be subtracted in extract_1d if one or more background regions are specified in the reference file.  There may be some extract_1d reference files in CRDS that do define background regions, but many do not.